pyscriptjs/examples/logic.py

```diff
@@ -0,0 +1,150 @@
+from js import console, document
+
+answer = Element("typing-text")
+num_first = ""
+num_second = ""
+sign_click = ""
+cal_type = ""
+numbers = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "."]
+sign = ["equal-to", "reverse", "ac", "percent", "plus-or-minus"]
+
+
+def numbers_clicked(args):
+    global num_first
```

Comment:
Why not put these on a class and move that to class scope rather than global?

The reason for this is that I'm not sure that using global is great in the context of PyScript where apps could have different namespaces or actually multiple scripts potentially referencing same globals objects. Just want to make sure we are not encouraging patterns that can be confusing in PyScript itself.

Reply:
+1 on refactoring this into a class. Whenever you find yourself using `global` a lot that's an indication you should be writing a class.

Reply:
@fpliger I have moved to class and refactored the codebase.